### DISCUSSION TITLE
Country combobox interaction

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#1420] [Q&A] [answered]
Hi Dsys, 
We are updating the 'Buy a number' page in the Console, and we noticed that the paste combobox makes the user erase the value to visualize all the options. In the legacy version this didn't happen, because the user could see all countries with a clic, even if a country was already selected. 
We wanted to know if this is possible to replicate in the paste version of this combobox, or if this limitation is due to other technical, accesibility, etc. issues. 

This is the video of the legacy combobox interaction: https://drive.google.com/file/d/1RYFWelH1NpwGWq_-PpPNmX_qE3JDKXNE/view?usp=sharing

**Comment by vankhoawin** [comment]
to give an interactive example, see the `useCombobox state hook` section in the Paste documentation: https://paste.twilio.design/components/combobox/#usecombobox-state-hook

1. Select a country (Andorra)
2. Focus the combobox again and see that the only option in the list is the selected country, Andorra

Is it possible to somehow show the entire option list? Just to give examples of other similar components in the open-source world:

https://react-select.com/home
https://material-ui.com/components/autocomplete/

**Comment by SiTaggart** [comment, marked as answer]
Hi 👋🏼 

This has come up before, and it is possible, we just make you do the work to achieve it.

Here's a link to a comment in thread about Combobox where the solution is touched on https://github.com/twilio-labs/paste/discussions/982#discussioncomment-144731, but you might find the whole thread useful, especially if you're an engineer.

Again, the documentation on Combobox will eventually get updated with some of these use cases spelled out, so folks don't have to figure so much out for themselves. It is a stage 1 component in terms of accompanying assets, docs and specs.
